draft-zip-hybrid-deferred-devfund.md

```diff
@@ -0,0 +1,413 @@
+  contribute to Zcash. Allowing the Dev Fund to expire, or putting 100% into a
+  lockbox, would disproportionally impact grant recipients. This hybrid
+  approach promotes decentralization and the growth of independent development
+  teams.
+
+* **Moving Towards a Non-Direct Funding Model**: There is strong community
+  support for a non-direct Dev Fund funding model.  Allocating funds to a
+  Hybrid Deferred Dev Fund Lockbox gives the community time to research and
+  implement such a model while continuing to fund grant recipients.
+
+
+## Requirements
+
+**In-Protocol Lockbox Creation**: The ZIP must define the creation of an
+in-protocol lockbox where 20% of the Zcash block reward will be allocated
+post-November 2024 halving. This lockbox will securely store funds until a
+disbursement mechanism is established in a future ZIP.
+
+**Decentralization**: The lockbox must be designed with the goal of future
+decentralization in mind. While the disbursement mechanism will be defined
+later, the lockbox should support the principles of decentralization to ensure
+future community involvement and project-specific funding.
+
+**Regulatory Considerations**: The creation of the lockbox should minimize
+regulatory risks by avoiding direct funding of specific organizations. The
+design should ensure compliance with applicable laws and regulations to support
+the long-term sustainability of the funding model.
+
+**Efficient Allocation Preparation**: The lockbox must be prepared to allocate
+resources efficiently once the disbursement mechanism is defined. This includes
+ensuring that funds are readily available for future projects and not tied up
+in organizational overhead.
+
+**Security of Funds**: The lockbox must implement robust security measures to
+protect against unauthorized access or misuse of funds. This includes ensuring
+the integrity of the future decentralized voting/signature mechanism.
+
+**Future Adaptability**: The lockbox design should be flexible and adaptable to
+accommodate the disbursement mechanism defined in a future ZIP. This ensures
+that the system can evolve based on feedback and new developments in the Zcash
+ecosystem.
+
+**Non-Direct Funding Model**: This ZIP recognizes there is strong community
+support for a non-direct funding model.  As such, ECC and ZF MUST collaborate
+with the Zcash community to research and explore the establishment of a
+non-direct funding model. The research should consider potential designs as
+well as possible legal and regulatory risks.
+
+**ZCG Slice**: The ZCG slice should encourage decentralization and support
+independent teams dedicated to Zcash. Major funding decisions should be based,
+to the extent feasible, on inputs from domain experts and pertinent
+stakeholders.
+
+**Value of ZEC**: In case the value of ZEC jumps, ZCG should not wastefully use
+excessive amounts of funds. Conversely, given market volatility and eventual
+halvings, it is desirable to create rainy-day reserves.
```

Comment:
These sections should be in a process subsection of the specification. Requirements constrain the ZIP itself, not anyone "implementing" the ZIP.

Reply:
@str4d Apologies, but I don't fully understand your comment here. Can you clarify or (even better) would you be able to propose a change that I can review and accept?